Sage/carbon
# fix(dialog-full-screen): fix content ref forwarding - FE-4062

### Proposed behaviour
This PR fixes issue with content ref not being forwarded in DialogFullScreen

Fixes #3980

### Checklist



### Testing instructions
https://codesandbox.io/s/heuristic-paper-vc6zn

Open DialogFullScreen and check that `<div ....` has been logged in console
